python/kfserving/kfserving/storage.py

```diff
@@ -63,6 +64,10 @@ def download(uri: str, out_dir: str = None) -> str:
             return Storage._download_local(uri, out_dir)
         elif re.search(_URI_RE, uri):
             return Storage._download_from_uri(uri, out_dir)
+        elif uri.startswith(MODEL_MOUNT_DIRS):
```

Comment:
for MMS, we do not even run the storage initializer right?

Reply:
The sklearn server also uses storage.py to download models. This change it to make it compatible with MMS. Otherwise, the sklearn server container will crash.

Reply:
hmm I thought it should hit line 52.

Reply:
It will only hit line 52 if the storageUri starts with ```file://```, I guess that's for PVC?

Reply:
still wondering why `os.path.exists(uri)` did not evaluate to be true for MMS case.